routers.js

```diff
@@ -12,3 +14,35 @@ export default (App) => {
         </Router>
     );
 };
+
+class AppRoute extends Relay.Route {
+    static path = '/';
+
+    static queries = {
+        user: () => Relay.QL`query { user }`,
+    };
+    static routeName = 'LauncherRoute';
+}
+
+export default (App) => {
+    return (
+        <Router
+            history={ browserHistory }
+            createElement={createRelayContainer}>
```

Comment:
Don't know, but I'm removing all this code. What we need is much simpler. I believe this is a copy of an article I saw somewhere. Cause we don't have the widgets route.